src/Confluent.SchemaRegistry.Serdes.Json/JsonSerializerConfig.cs

```diff
@@ -64,13 +64,21 @@ public static class PropertyNames
             /// <summary>
             ///     Specifies whether or not the JSON serializer should use the latest subject
             ///     version for serialization.
-            ///     WARNING: There is no check that the latest schema is backwards compatible
-            ///     with the schema of the object being serialized.
             ///
             ///     default: false
             /// </summary>
             public const string UseLatestVersion = "json.serializer.use.latest.version";
 
+            /// <summary>
+            ///     Specifies whether or not the JSON serializer should check the backwards compatibility 
+            ///     with the latest schema of the subject.
+            ///     WARNING: There is no check that the latest schema is backwards compatible
```

Comment:
i don't think you want this warning here, you want it on UseLatestVersion, as that is where the user will need to know about is.

Reply:
I called this as a warning because if LatestCompatStrict config is ignored and set to default there can be issue with serializer sending any info with local class structure.

Having the warning on UseLatestVersion also makes sense.

Reply:
Migrating the Warning to UseLatestVersion.